### PR TITLE
Cu 82pe8j css integration navigation breadcrumbs

### PR DESCRIPTION
**Pull request by Hamza-A**
This branch aims to provide new navigation breadcrumbs within each pages to make them look similar to our wireframe

## Description
I began with a groundwork of removing all the business logics and css from all files in `src/pages` to move them in the component they're calling.

Secondly, I made sure that every child component contains a `<Wrapper />` with the same margin and padding. This is an important point because navigation breadcrumbs must be displayed at the same place in every page.

Finally, I've created a new component `<NavigationBreadcrumbs />` and its story, it contains 2 properties `title` and `breadcrumbs` which accepts `strings` only. By doing this, the component became configurable and can be adapted to each page.

## How Has This Been Tested?
I've added tests in each page that calls `<NavigationBreadcrumbs />` to check if it's displayed properly

## Screenshots
image
image
image
image
image

## Types of changes

- New feature (non-breaking change which adds functionality)

## Checklist:

- :green_circle: My code follows the code style of this project.
- :red_circle: My change requires a change to the documentation.
- :red_circle: I have updated the documentation accordingly.
- :green_circle: I have read the [**CONTRIBUTING**][contributing_file] document.
- :green_circle: I have added tests to cover my changes.
- :green_circle: All new and existing tests passed.

[contributing_file]: https://github.com/fewlinesco/connect/blob/master/CONTRIBUTING.md

### CODE REVIEW

**Review by Fuzznimp — COMMENTED** [review]
Before I review it further, could you replace the file path starting with `../` with the alias instead of the relative path?

**Comment by Hamza-A** [comment]
> Before I review it further, could you replace the file path starting with `../` with the alias instead of the relative path?

File: tests/pages/updateIdentity.test.tsx [hunk 1]
```diff
@@ -1,27 +1,28 @@
...
+import { GlobalStyle } from "@src/design-system/globals/globalStyle";
+import { lightTheme } from "@src/design-system/theme/lightTheme";
+import { useCookies } from "@src/hooks/useCookies";
+import UpdateIdentityPage from "@src/pages/account/logins/[type]/[id]/update";
+import * as fetchJson from "@src/utils/fetchJson";
 
 enableFetchMocks();
 
-jest.mock("../../src/config", () => {
+jest.mock("@src/config", () => {
```
Comment (Fuzznimp) [inline tests/pages/updateIdentity.test.tsx:25]: Hmm, I was wondering if this was working 😅.

Comment (Hamza-A) [reply tests/pages/updateIdentity.test.tsx:25]: It does 😜

File: src/components/LoginsPage.tsx [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+
+  return (
+    <Wrapper>
+      <Head>
+        <title>Connect Logins</title>
+      </Head>
+      <IdentitySection>
+        <NavigationBreadcrumbs
+          title="Logins"
+          breadcrumbs="Your emails, phones and social logins"
```
Comment (Fuzznimp) [inline src/components/LoginsPage.tsx:53]: This shouldn't be a breadcrumb, but just a small header. Breadcrumbs represent a progression.

File: src/components/LoginsPage.tsx [hunk 3]
```diff
@@ -0,0 +1,175 @@
+import Head from "next/head";
```
Comment (Fuzznimp) [inline src/components/LoginsPage.tsx:1]: This is not a page. You could name this `Identities`, and `src/pages/account/logins/index.tsx` should be called `LoginsPage`.

File: src/components/display/fewlines/NavigationBreadcrumbs/NavigationBreadcrumbs.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import React from "react";
```
Comment (Fuzznimp) [inline src/components/display/fewlines/NavigationBreadcrumbs/NavigationBreadcrumbs.tsx:1]: The title should be independent of the breadcrumbs :).

File: src/components/LoginsPage.tsx (hunk 2 above)
Comment (Hamza-A) [reply src/components/LoginsPage.tsx]: OK fixed 2762b34

File: src/components/display/fewlines/UpdateIdentityForm.tsx [hunk 5]
```diff
@@ -2,19 +2,37 @@ import { useRouter } from "next/router";
...
   const [identity, setIdentity] = React.useState("");
   const router = useRouter();
+  const { value } = currentIdentity;
 
   return (
     <Wrapper>
+      <NavigationBreadcrumbs
+        title="Logins"
+        breadcrumbs={`${
+          currentIdentity.type === IdentityTypes.EMAIL.toLowerCase()
```
Comment (Fuzznimp) [inline src/components/display/fewlines/UpdateIdentityForm.tsx:28]: We should really settle about using `IdentityTypes` vs `ReceivedIdentityTypes`.

I really don't like that we have 2 sources of verity for identity types. Also, the end goal will be to have `IdentityTypes` type coming from the SDK.

I'll let it like this for now, but when my PR is merged, we'll need to think about this.

File: src/components/LoginsPage.tsx (hunk 2 above)
Comment (Fuzznimp) [reply src/components/LoginsPage.tsx]: Well, header means title in this case. So you can't do it like this :).

```jsx
<h1>Logins</h1>
<h2>Your emails, phones and social logins</h2>
```

If you need a wrapper, you can call it `HeadersWrapper`.

If you look at this:
image

First line is an `h1`, second is an `h3` and third line is an `h2`.

File: src/components/LoginsPage.tsx (hunk 3 above)
Comment (Hamza-A) [reply src/components/LoginsPage.tsx]: OK fixed ce50f4b

File: src/components/LoginsPage.tsx (hunk 2 above)
Comment (Hamza-A) [reply src/components/LoginsPage.tsx]: Actually if you double check my commit, I did use a `<SmallHeader />` wrapper in which title and subtitle are nested.
As a matter of fact, first line in an `h1` second line is a `p` with a `font-size` of `1.4rem` and third line is an `h3`

File: src/components/display/fewlines/NavigationBreadcrumbs/NavigationBreadcrumbs.tsx (hunk 4 above)
Comment (Hamza-A) [reply src/components/display/fewlines/NavigationBreadcrumbs/NavigationBreadcrumbs.tsx:1]: OK fixed 6b53cce 2d72a91

File: src/components/display/fewlines/IdentityOverview.tsx [hunk 6]
```diff
@@ -2,12 +2,12 @@ import Link from "next/link";
 import React from "react";
 import styled from "styled-components";
 
-import { Identity, ReceivedIdentityTypes } from "../@types/Identity";
-import { AwaitingValidationBadge } from "./display/fewlines/AwaitingValidationBadge/AwaitingValidationBadge";
-import { Box } from "./display/fewlines/Box/Box";
-import { Button, ButtonVariant } from "./display/fewlines/Button/Button";
-import { NavigationBreadcrumbs } from "./display/fewlines/NavigationBreadcrumbs/NavigationBreadcrumbs";
-import { PrimaryBadge } from "./display/fewlines/PrimaryBadge/PrimaryBadge";
+import { Identity, ReceivedIdentityTypes } from "../../../@types/Identity";
```
Comment (Fuzznimp) [inline src/components/display/fewlines/IdentityOverview.tsx:5]:
```suggestion
import { Identity, ReceivedIdentityTypes } from "@src/@types/Identity";
```

File: src/components/display/fewlines/NavigationBreadcrumbs/NavigationBreadcrumbs.tsx [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+};
+
+const Wrapper = styled.div`
+  padding: 0 0 ${({ theme }) => theme.spaces.component.s};
+
+  p {
+    font-weight: ${({ theme }) => theme.fontWeights.light};
+    font-size: ${({ theme }) => theme.fontSizes.s};
+  }
+`;
```
[the remark this answers is not in the document]
Comment (Hamza-A) [reply src/components/display/fewlines/NavigationBreadcrumbs/NavigationBreadcrumbs.tsx]: OK fixed 7de9201 9dae164

File: src/components/display/fewlines/IdentityOverview.tsx (hunk 6 above)
Comment (Hamza-A) [reply src/components/display/fewlines/IdentityOverview.tsx]: OK fixed eaef32b

File: tests/pages/IdentityValidationPage.test.tsx [hunk 8]
```diff
@@ -38,15 +43,59 @@ jest.mock("../../src/config", () => {
   };
 });
 
-describe("IdentityValidationForm", () => {
+describe("IdentityValidationPage", () => {
   const eventId = "foo";
 
-  test("it should display properly an input and 3 buttons for emails", () => {
+  test("it shoud display navigation breadcrumbs properly for emails", () => {
```
Comment (Fuzznimp) [inline tests/pages/IdentityValidationPage.test.tsx:49]:
```suggestion
  test("it should display navigation breadcrumbs properly for emails", () => {
```

File: tests/pages/ShowIdentityPage.test.tsx [hunk 9]
```diff
@@ -67,12 +71,55 @@ describe("ShowIdentity", () => {
...
 
+  const phoneIdentity: Identity = {
+    id: "81z343c1-530b-4982-878d-33f0def6a7cf",
+    primary: false,
+    status: "unvalidated",
+    type: ReceivedIdentityTypes.PHONE,
+    value: "0722443311",
+  };
+
+  test("it shoud display navigation breadcrumbs properly for emails", () => {
```
Comment (Fuzznimp) [inline tests/pages/ShowIdentityPage.test.tsx:82]:
```suggestion
  test("it should display navigation breadcrumbs properly for emails", () => {
```

File: tests/pages/ShowIdentityPage.test.tsx [hunk 10]
```diff
@@ -67,12 +71,55 @@ describe("ShowIdentity", () => {
...
+
+    const navigationBreadCrumbs = component.find(NavigationBreadcrumbs);
+    expect(navigationBreadCrumbs).toHaveLength(1);
+
+    expect(
+      navigationBreadCrumbs.contains(<Breadcrumbs>Email address</Breadcrumbs>),
+    ).toEqual(true);
+  });
+
+  test("it shoud display navigation breadcrumbs properly for phones", () => {
```
Comment (Fuzznimp) [inline tests/pages/ShowIdentityPage.test.tsx:100]:
```suggestion
  test("it should display navigation breadcrumbs properly for phones", () => {
```

File: tests/pages/UpdateIdentityPage.test.tsx [hunk 11]
```diff
@@ -51,6 +55,52 @@ describe("UpdateIdentityPage", () => {
...
 
+  const phoneIdentity: Identity = {
+    id: "htg6dcc1-530b-4982-878d-33f0def6a7cf",
+    primary: false,
+    status: "validated",
+    type: ReceivedIdentityTypes.PHONE,
+    value: "0788992255",
+  };
+
+  test("it shoud display navigation breadcrumbs properly for emails", () => {
```
Comment (Fuzznimp) [inline tests/pages/UpdateIdentityPage.test.tsx:66]:
```suggestion
  test("it should display navigation breadcrumbs properly for emails", () => {
```

File: tests/pages/UpdateIdentityPage.test.tsx [hunk 12]
```diff
@@ -51,6 +55,52 @@ describe("UpdateIdentityPage", () => {
...
+    const navigationBreadCrumbs = component.find(NavigationBreadcrumbs);
+    expect(navigationBreadCrumbs).toHaveLength(1);
+    expect(
+      navigationBreadCrumbs.contains(
+        <Breadcrumbs>Email address | edit</Breadcrumbs>,
+      ),
+    ).toEqual(true);
+  });
+
+  test("it shoud display navigation breadcrumbs properly for phones", () => {
```
Comment (Fuzznimp) [inline tests/pages/UpdateIdentityPage.test.tsx:85]:
```suggestion
  test("it should display navigation breadcrumbs properly for phones", () => {
```

File: tests/pages/IdentityValidationPage.test.tsx [hunk 13]
```diff
@@ -38,15 +43,59 @@ jest.mock("../../src/config", () => {
...
+    const navigationBreadCrumbs = component.find(NavigationBreadcrumbs);
+    expect(navigationBreadCrumbs).toHaveLength(1);
+    expect(
+      navigationBreadCrumbs.contains(
+        <Breadcrumbs>Email address | validation</Breadcrumbs>,
+      ),
+    ).toEqual(true);
+  });
+
+  test("it shoud display navigation breadcrumbs properly for phones", () => {
```
Comment (Fuzznimp) [inline tests/pages/IdentityValidationPage.test.tsx:71]:
```suggestion
  test("it should display navigation breadcrumbs properly for phones", () => {
```

File: tests/pages/IdentityValidationPage.test.tsx (hunk 8 above)
Comment (Hamza-A) [reply tests/pages/IdentityValidationPage.test.tsx]: OK fixed 2433c2a

File: tests/pages/ShowIdentityPage.test.tsx (hunk 9 above)
Comment (Hamza-A) [reply tests/pages/ShowIdentityPage.test.tsx]: OK fixed 2433c2a

File: tests/pages/ShowIdentityPage.test.tsx (hunk 10 above)
Comment (Hamza-A) [reply tests/pages/ShowIdentityPage.test.tsx]: OK fixed 2433c2a

File: tests/pages/UpdateIdentityPage.test.tsx (hunk 11 above)
Comment (Hamza-A) [reply tests/pages/UpdateIdentityPage.test.tsx]: OK fixed 2433c2a

File: tests/pages/UpdateIdentityPage.test.tsx (hunk 12 above)
Comment (Hamza-A) [reply tests/pages/UpdateIdentityPage.test.tsx]: OK fixed 2433c2a

File: tests/pages/IdentityValidationPage.test.tsx (hunk 13 above)
Comment (Hamza-A) [reply tests/pages/IdentityValidationPage.test.tsx]: OK fixed 2433c2a

**Review by Fuzznimp — APPROVED** [review]

**Pull request merged**